erpnext/stock/report/stock_ageing/stock_ageing.py

```diff
@@ -7,13 +7,15 @@
 from frappe.utils import date_diff, flt
 from six import iteritems
 from erpnext.stock.doctype.serial_no.serial_no import get_serial_nos
+from erpnext.compliance.doctype.package_tag.package_tag import get_package_tag_batch_details
 
 def execute(filters=None):
 
 	columns = get_columns(filters)
 	item_details = get_fifo_queue(filters)
 	to_date = filters["to_date"]
 	_func = lambda x: x[1]
+	package = get_package_tag_batch_details()
```

Comment:
```suggestion
	package_tags = get_package_tag_batch_details()
```
- Rename it at multiple places